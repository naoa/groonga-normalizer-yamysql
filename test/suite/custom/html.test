register normalizers/yamysql

normalize NormalizerYaMySQLHtml "<p>これは<b>テスト</b>です。</p>" WITH_TYPES|WITH_CHECKS

