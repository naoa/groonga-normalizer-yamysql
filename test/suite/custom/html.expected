register normalizers/yamysql
[[0,0.0,0.0],true]
normalize NormalizerYaMySQLHtml "<p>これは<b>テスト</b>です。</p>" WITH_TYPES|WITH_CHECKS
[
  [
    0,
    0.0,
    0.0
  ],
  {
    "normalized": "   これは   テスト    です。    ",
    "types": [
      "unknown",
      "unknown",
      "unknown",
      "hiragana",
      "hiragana",
      "hiragana",
      "unknown",
      "unknown",
      "unknown",
      "katakana",
      "katakana",
      "katakana",
      "unknown",
      "unknown",
      "unknown",
      "unknown",
      "hiragana",
      "hiragana",
      "symbol",
      "unknown",
      "unknown",
      "unknown",
      "unknown"
    ],
    "checks": [
      1,
      1,
      1,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      1,
      1,
      1,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      1,
      1,
      1,
      1,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      1,
      1,
      1,
      1
    ]
  }
]
